src/model/constants.rs

```diff
@@ -1,14 +1,21 @@
-
+// Model constants
 pub static MULTIPLIER: f64 = 45.0;
 pub static SIGMA: f64 = (25.0 / 5.0) * MULTIPLIER;
```

Comment:
```suggestion
pub static SIGMA: f64 = 5.0 * MULTIPLIER;
```
Mostly because we're no longer sticking to the default of 25, so it's good for future reference not to tie it to that number.